### PR TITLE
fix: fee charge token create tx and also transaction hashing

### LINKED ISSUE #11526: Replay events from round 162444678 and validate record files for differential testing - 15 min
Replay events from round 162444678 and validate record files for differential testing - 15 min
issues
Mismatched field counts  (expected but not find [amount]) between expected contractID {
Mismatched values, expected '<ByteString@64e92d61 size=48 contents="\301M\020\254\fA\277\354\r\360m\200\375\204\220,tRy\201\363G\r\3764>:\210\373\261\177a\032\357\206\323\032\350%\343Z\002\363\301\344\343\215*">', got '<ByteString@7882c44a size=48 contents="\370*\355e\224j=-c\275&L\362\016\200\324\2761;\313\001\366\312\235\004 Ru>#\311N\270Ng:\2759\353\372u\361d\255\250dYq">' - Matching field 'transactionHash'  ==> expected: <<ByteString@64e92d61 size=48 contents="\301M\020\254\fA\277\354\r\360m\200\375\204\220,tRy\201\363G\r\3764>:\210\373\261\177a\032\357\206\323\032\350%\343Z\002\363\301\344\343\215*">> but was: <<ByteString@7882c44a size=48 contents="\370*\355e\224j=-c\275&L\362\016\200\324\2761;\313\001\366\312\235\004 Ru>#\311N\270Ng:\2759\353\372u\361d\255\250dYq">>

### PR DESCRIPTION
**Pull request by povolev15**
Fix both issues
Mismatched field counts (expected but not find [amount]) between expected contractID {

Mismatched values, expected '<ByteString@64e92d61 size=48 contents="\301M\020\254\fA\277\354\r\360m\200\375\204\220,tRy\201\363G\r\3764>:\210\373\261\177a\032\357\206\323\032\350%\343Z\002\363\301\344\343\215*">', got '<ByteString@7882c44a size=48 contents="\370*\355e\224j=-c\275&L\362\016\200\324\2761;\313\001\366\312\235\004 Ru>#\311N\270Ng:\2759\353\372u\361d\255\250dYq">' - Matching field 'transactionHash' ==> expected: <<ByteString@64e92d61 size=48 contents="\301M\020\254\fA\277\354\r\360m\200\375\204\220,tRy\201\363G\r\3764>:\210\373\261\177a\032\357\206\323\032\350%\343Z\002\363\301\344\343\215*">> but was: <<ByteString@7882c44a size=48 contents="\370*\355e\224j=-c\275&L\362\016\200\324\2761;\313\001\366\312\235\004 Ru>#\311N\270Ng:\2759\353\372u\361d\255\250dYq">>
**Related issue(s)**:

Fixes #11526

### CODE REVIEW

**Review by tinker-michaelj — APPROVED** [review]
LGTM, ty @povolev15 !

File: hedera-node/test-clients/src/rcdiff/java/com/hedera/services/rcdiff/RcDiff.java [hunk 1]
```diff
@@ -182,20 +186,17 @@ private String readableDiff(@NonNull final DifferingEntries diff) {
...
+                    .append("➡️ but was ")
                     .append(Objects.requireNonNull(diff.secondEntry()).consensusTime());
         } else if (firstEncounteredDifference == TRANSACTION_RECORD_MISMATCH
                 || firstEncounteredDifference == TRANSACTION_MISMATCH) {
             sb.append("\nFor body,\n")
                     .append(Objects.requireNonNull(diff.firstEntry()).body());
-            sb.append("➡️  Expected Receipt ")
-                    .append(Objects.requireNonNull(diff.firstEntry())
-                            .transactionRecord()
-                            .getReceipt())
```
Comment (kimbor) [inline hedera-node/test-clients/src/rcdiff/java/com/hedera/services/rcdiff/RcDiff.java:194]: Was it intentional to drop getReceipt() from here and below?

Comment (povolev15) [reply hedera-node/test-clients/src/rcdiff/java/com/hedera/services/rcdiff/RcDiff.java:194]: yes it will include both and not just the reciept

Comment (povolev15) [reply hedera-node/test-clients/src/rcdiff/java/com/hedera/services/rcdiff/RcDiff.java:194]: The record include the receipt

**Review by mhess-swl — APPROVED** [review]

**Pull request merged**